proto/ibc/applications/perm/v1/types.proto

```diff
@@ -6,8 +6,12 @@ option go_package = "github.com/initia-labs/initia/x/ibc/perm/types";
 
 // PermissionedRelayer is used to specifiy the permissioned relayer for
 // the specific port-id:channel-id pair.
-message PermissionedRelayer {
+message PermissionedRelayersSet {
   string port_id    = 1;
   string channel_id = 2;
-  string relayer    = 3;
+  PermissionedRelayerList relayer_list    = 3;
```

Comment:
we can keep this as just repeated string

and use `PermissionedRelayerList` as a wrapper when we store it